static/src/components/fieldsconfig/consensus_config.vue

```diff
@@ -105,17 +105,17 @@ export default {
     ...mapMutations(['updateConsensusConfig']),
 
     _isIntegerNumeric: function (n) {
-        var _n = Number(n);
+        var _n = parseInt(n, 10);
         return Math.floor(_n) === _n;
     },
 
     _write: function () {
-        if (!this._isIntegerNumeric(this.threshold) || this.threshold <= 50 ||
-                this.threshold > 100) {
+        if (!this._isIntegerNumeric(this.consensusThreshold) || this.consensusThreshold <= 50 ||
```

Comment:
Just double checking: do you need to cast e.g. `this.consensusThreshold` to int here?  Like you do here: https://github.com/bloomberg/pybossa-default-theme/pull/118/files#diff-1e8c3142d2cb9be9f01400dcc7555c71R134

Reply:
I didn't cast it to int here, just check whether the input contains letters or is float number.